dbx/encrypt-fields.rst

```diff
@@ -11,58 +11,73 @@ In-Use Encryption
 Overview
 --------
 
-You can encrypt fields in a document using a set of features called
-**in-use encryption**.
-
-In-use encryption enables your client applications to encrypt data
-*before* sending it to MongoDB, and to query documents with encrypted fields.
+You can use the {+driver-short+} to encrypt document fields by using a
+set of features called **in-use encryption**. In-use encryption allows
+your application to encrypt data *before* sending it to MongoDB
+and query documents with encrypted fields.
 
 Because the driver encrypts the data before sending it to MongoDB, only
-your configured client applications can decrypt the data. Only applications
-using the driver with access to your encryption keys can access the decrypted,
-plaintext data. Should you have unauthorized access to your database, an
-attacker could only see the encrypted, ciphertext data.
+your configured applications can decrypt the data. Only applications
```

Comment:
Suggestion:

I think this paragraph doesn't build up the knowledge the reader might need to follow what these features accomplish without making some assumptions. I think continuing the description of in-use encryption could be helpful and then providing some details on how it works with MongoDB and your application.

Here's the information that I think needs to be presented in this paragraph:

1) In-use encryption prevents anyone from viewing the decrypted, plaintext data directly from the database or when the data is in-transit (on the network).

2) To set up your application to use in-use encryption, you must create encryption keys that only your application has access to

3) Your application must use the in-use encryption features and the encryption keys to decrypt the encrypted data

4) If an attacker gets access directly to your database, they cannot see any decrypted data

Let me know if you need any help with conveying these.